lua/neotest-golang/lib/cmd.lua

```diff
@@ -9,13 +9,15 @@ local json = require("neotest-golang.lib.json")
 local M = {}
 
 function M.golist_data(cwd)
-  -- call 'go list -json ./...' to get test file data
+  -- call 'go list -json {go_list_args...} ./...' to get test file data
   local go_list_command = {
     "go",
     "list",
     "-json",
-    "./...",
   }
+  -- combine base command, user args and packages(./...)
+  vim.list_extend(vim.list_extend(go_list_command, options.get().go_list_args or {}), { "./..." })
```

Comment:
To fix the formatting issues in CI:

```suggestion
    vim.list_extend(
    vim.list_extend(go_list_command, options.get().go_list_args or {}),
    { "./..." }
  )
```